deploy-board/deploy_board/templates/configs/new_capacity.html

```diff
@@ -61,11 +61,11 @@ <h4 class="panel-title pull-left pointer-cursor">Basic Settings</h4>
                 </div>
                 <label-input label="Capacity" placeholder="# of instances" v-model="instanceCount"></label-input>
                 <hosttype-select label="Host Type" title="Compute Capability of the host" v-model="selectedHostTypeValue" v-bind:selectoptions="hostTypeOptions" selectsearch="true"
-                    ></hosttype-select>               
+                    ></hosttype-select>
                 <label-select label="Security Zone" title="Security zone to control inbound/outboud traffic" v-model="selectedSecurityZoneValue" v-bind:selectoptions="securityZoneOptions"></label-select>
                 <placements-select label="Placements" title="Placements" v-bind:assignpublicip="assignPublicIP"
                 v-bind:selectoptions="placements" v-on:assignpublicipclick="selectpublicip"></placements-select>
-                <accessrole-input label="Access Role" title="access_role for host access" v-model="accessRole" 
+                <accessrole-input label="Access Role" title="access_role for host access" v-model="accessRole"
                     showhelp="true" v-on:helpclick="accessRoleHelpClick"></accessrole-input>
                 <accessrole-help v-show="showAccessRoleHelp" v-bind:data="accessRoleHelpText" v-bind:accessrolelist="accessRoleList"></accessrole-help>
```

Comment:
https://github.com/pinterest/teletraan/blob/5048cc2911bdd42b364810f5498c024546d0f9bd/deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js#L330

I see `accessRoleList` being used as a link here. Is that expected?

Reply:
Yes. Tag @rfleur01 for naming convention awareness.

Reply:
Acknowledged. It's a link from teletraanconfigs

Reply:
maybe it can be renamed to validAccessRoleURL?

Reply:
yes that's better